### PR TITLE
Check waiter panel tests

### PR DESCRIPTION
**Pull request by AhnessP**

### CODE REVIEW

File: EasyRestProjectNetTeam2/EasyRestTests/CheckWaitersTests.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        {
+            waiterPanelPage.WaitClickInProgressTab(dataModel.TimeToWait);
+            waiterPanelPage.WaitClickArrowDownButton(dataModel.TimeToWait);
+            waiterPanelPage.WaitClickCloseOrderButton(dataModel.TimeToWait);
+            waiterPanelPage.WaitConfirmationWindowAndCheckIfDisplayed(dataModel.TimeToWait);
+            var expected = waiterPanelPage.IsConfirmationWindowExist();
+            Assert.IsTrue(expected, "Waiter is unable to close order in progress");
+        }
+
+        public override void TearDown()
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckWaitersTests.cs:52]: should you add `[Teardown]` attribute?

Comment (AhnessP) [reply EasyRestProjectNetTeam2/EasyRestTests/CheckWaitersTests.cs:52]: fixed

**Review by dmatsev — CHANGES_REQUESTED** [review]
Please fix the solution.

File: EasyRestProjectNetTeam2/EasyRestTests/CheckWaitersTests.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            DatabaseManager.SendNonQuery(queryDataModel.InsertOrderInAssignedWaiterStatus);
+            signInPage = GetSignInPage();
+            homePage = GetHomePage();
+            baseSignIn = new BaseSignIn(signInPage, homePage);
+            baseSignIn.SignIn(dataModel.WaiterEmail, dataModel.ShortPassword);
+            waiterPanelPage = GetWaiterPanelPage();
+        }
+
+        [Test]
+
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckWaitersTests.cs]: please remove empty lines

Comment (AhnessP) [reply EasyRestProjectNetTeam2/EasyRestTests/CheckWaitersTests.cs]: fixed

**Review by dmatsev — APPROVED** [review]

**Review by SerhiiTarasiuk — APPROVED** [review]

**Pull request merged**